.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,57 +1,131 @@
+
+  Marking Unfinished PRs for Review or as a Work in Progress (WIP)
+  https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
+-->
+
+**Which is the sponsoring or primary SIG associated with this PR?**
+<!--
+Label the sponsoring or primary SIG associated with this PR.
+
+For reference on available SIGs, you can find more details at:
+https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.md#labels-that-apply-to-all-repos-for-both-issues-and-prs
+
+Add only ONE sig here via a /sig-primary <name> entry.
 -->
+/sig-primary ???
+
+> DISCUSSION REQUIRED
+>
+> Should we use the above pattern and update the label prow plugin ? https://github.com/kubernetes/test-infra/tree/master/prow/plugins/label. Logic would have to ensure that there is only a single primary sig associated. This metadata can be leveraged by other processes outside of the release notes.
+> 
+> or, if this is ONLY relevant to release notes then, we can add here as a `` ```primary-sig`` code block.
 
 **What type of PR is this?**
-> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
+<!-- 
+Label this pull request according to what type of issue you are addressing, especially if this is a release targeted pull request. 
+
+For reference on required PR labels, you can find more details at:
+https://git.k8s.io/community/contributors/devel/sig-release/release.md#issuepr-kind-label
+
+Add only ONE of the following /kind <type> entries.
+
+/kind api-change
+/kind bug
+/kind cleanup
+/kind design
+/kind documentation
+/kind failing-test
+/kind feature
+/kind flake
+-->
+/kind ???
+
+> DISCUSSION REQUIRED
 >
-> /kind api-change
-> /kind bug
-> /kind cleanup
-> /kind design
-> /kind documentation
-> /kind failing-test
-> /kind feature
-> /kind flake
+> Should we include a kind type of `deprecation`? And should we then include a `` ```depracation-notes`` code block to provide specific detailed notes around the depracation.
+> 
+> Should we only allow a single kind to be specified? There are a number of examples where multiple kind values make sense.
 
 **What this PR does / why we need it**:
+<!--
+Provide a description on what this PR does and/or why it is required.
+-->
+Description here ...
 
 **Which issue(s) this PR fixes**:
 <!--
-*Automatically closes linked issue when PR is merged.
-Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
-_If PR is about `failing-tests or flakes`, please post the related issues/tests in a comment and do not use `Fixes`_*
+Add references to issues that are fixed by this PR by using one or more of the following templates:
+
+Fixes #<issue number>
+Fixes <issue url>
+
+NOTE: The linked issue will automatically be closed when the PR is merged.
+
+If the PR is about failing-tests or flakes, please post the related issues/tests in a comment and do not use the Fixes templates above.
+-->
+Fixes #???
```

Comment:
Would prefer to keep this as-is because adding `???` would require the author to remove them first :grimacing:

Reply:
💯 In general, the less people have to change / fill out. The more likely they are to use it appropriately. This applies to the other areas below as well.

Reply:
Agree on minimising work for folks. Not all PRs fix an issue, they may reference one or more issues, or reference an issue in another repo. Should the `Fixes #` text not be removed then too, and the comments section guide a PR submitter on how to format.

Or do the overwhelming majority of PRs relate to fixing an issue, which would then make sense to keep the `Fixes #` text in, minus the `???`.

Reply:
Good point, @paulbouwer!

While not all PRs might "fix" an issue, I'd prefer to keep `Fixes #` because I've seen some cases where people forget to add this line and the issue needs to explicitly closed later. However, in case anyone has data around cases where this has led to issues being closed unexpectedly, we can consider removing it. :)

Reply:
I'll remove the `???`.